ssl/statem/statem_srvr.c

```diff
@@ -2402,7 +2404,7 @@ int tls_construct_server_hello(SSL_CONNECTION *s, WPACKET *pkt)
                                           : SSL_EXT_TLS1_2_SERVER_HELLO),
                                   NULL, 0)) {
         /* SSLfatal() already called */
-        return 0;
+        return CON_FUNC_SUCCESS;
```

Comment:
Is this right? 0 is converted to ERROR elsewhere.

Reply:
Oops. No. Fixed.